docs/README.md

```diff
@@ -53,6 +54,17 @@ We require all contributors to acknowledge the [Forge Contributor License Agreem
 Please ensure you have a valid email address associated with your GitHub account to do this. If you have previously 
  signed it, you should be OK.
 
+### ❓ Frequently asked questions (FAQ)
+
+**Q : Which Java version is required ?**  
+→ Forge works with **JDK 17+**.
```

Comment:
This doesn't tell the whole story. Forge is a layer on top of Minecraft, so it's going to need whatever Java version the game version runs on. This is Java 8 for 1.16-, Java 16 for 1.17, Java 17 for 1.18-1.20.4, and Java 21 for 1.20.6 onwards.